wkconnect/routes/datasets/meshes.py

```diff
@@ -10,7 +12,66 @@
 async def get_meshes(
     request: Request, organization_name: str, dataset_name: str, layer_name: str
 ) -> response.HTTPResponse:
-    del organization_name
-    del dataset_name
-    del layer_name
-    return response.json([])
+    (backend_name, dataset) = request.app.ctx.repository.get_dataset(
+        organization_name, dataset_name
+    )
+    backend = request.app.ctx.backends[backend_name]
+
+    try:
+        return response.json(await backend.get_meshes(dataset, layer_name))
+    except NotImplementedError:
+        return response.empty(status=501)
+
+
+@meshes.route(
+    "/<organization_name>/<dataset_name>/layers/<layer_name>/meshes/chunks",
+    methods=["POST"],
+)
+async def get_mesh_chunks(
+    request: Request, organization_name: str, dataset_name: str, layer_name: str
+) -> response.HTTPResponse:
+    request_body = request.json
+
+    (backend_name, dataset) = request.app.ctx.repository.get_dataset(
+        organization_name, dataset_name
+    )
+    backend = request.app.ctx.backends[backend_name]
+
+    segment_id = request_body["segmentId"]
+    try:
+        chunks = await backend.get_chunks_for_mesh(
+            dataset, layer_name, "mesh", segment_id
+        )
+        if chunks is None:
+            return response.empty(status=404)
+        return response.json(chunks)
+    except NotImplementedError:
+        return response.empty(status=501)
+
+
+@meshes.route(
+    "/<organization_name>/<dataset_name>/layers/<layer_name>/meshes/chunks/data",
+    methods=["POST"],
+)
+async def get_mesh_chunk_data(
```

Comment:
the routes may need `@authorized(AccessRequest.read_dataset)` so the authentication is checked